database/factories/CommentFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\Comment;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<Comment>
+ */
+class CommentFactory extends Factory
+{
+    protected $model = Comment::class;
+
+    public function definition(): array
+    {
+        $user = User::inRandomOrder()->first();
+
+        return [
+            'Payload' => $this->faker->paragraph,
+            'Submitted' => $this->faker->dateTimeBetween('-1 year', 'now'),
+            'Edited' => $this->faker->dateTimeBetween('now', '+1 year'),
```

Comment:
Can we give this random odds of being set? I did 1/12 odds in another branch:

https://github.com/RetroAchievements/RAWeb/blob/master/database/factories/CommentFactory.php#L20